dipy/tracking/fast_tracking.pyx

```diff
@@ -0,0 +1,256 @@
+# cython: boundscheck=False
+# cython: initializedcheck=False
+# cython: wraparound=False
+# cython: Nonecheck=False
+
+from libc.stdio cimport printf
+
+cimport cython
+from cython.parallel import prange
+import numpy as np
+cimport numpy as cnp
+
+from dipy.core.interpolation cimport trilinear_interpolate4d_c
+from dipy.direction.pmf cimport PmfGen
+from dipy.tracking.stopping_criterion cimport StoppingCriterion
+from dipy.utils cimport fast_numpy
+
+from dipy.tracking.stopping_criterion cimport (StreamlineStatus,
+                                               StoppingCriterion,
+                                               TRACKPOINT,
+                                               ENDPOINT,
+                                               OUTSIDEIMAGE,
+                                               INVALIDPOINT,
+                                               VALIDSTREAMLIME,
+                                               INVALIDSTREAMLIME)
+from dipy.tracking.tracker_parameters cimport (TrackerParameters, 
+                                               func_ptr, 
+                                               SUCCESS, 
+                                               FAIL)
+
+from nibabel.streamlines import ArraySequence as Streamlines
+
+from libc.stdlib cimport malloc, free
+from libc.string cimport memcpy
+from libc.math cimport floor, ceil
+from libc.stdio cimport printf
+
+from posix.time cimport clock_gettime, timespec, CLOCK_REALTIME
```

Comment:
`posix.time` : Unix-specific header file and is not available in the standard library of Windows, which is what Visual Studio targets.

That's why it fails on windows.